cirq/google/common_serializers.py

```diff
@@ -418,6 +418,69 @@ def _convert_physical_z(op: ops.Operation, proto: v2.program_pb2.Operation):
         args=[]),
 ]
 
+
+#
+# FSim serializer
+# Only allows sqrt_iswap, its inverse, identity, and sycamore
+#
+def _can_serialize_limited_fsim(theta: float, phi: float):
+    # Identity
+    if _near_mod_2pi(theta, 0) and _near_mod_2pi(phi, 0):
+        return True
+    # sqrt ISWAP and inverse sqrt ISWAP
+    if _near_mod_2pi(abs(theta), np.pi / 4) and _near_mod_2pi(phi, 0):
+        return True
```

Comment:
There's a subtlety here with periodicity and absolute value; we want to accept theta = pi/4 and -pi/4, but also theta = 7pi/4 (which is equal to -pi/4 mod 2pi). The current code will not accept that. I think we should do:
```python
if (_near_mod_2pi(theta, np.pi / 4) or _near_mod_2pi(theta, -np.pi / 4)) and _near_mod_2pi(phi, 0):
    ...
```
Could also split these into separate if clauses and label them explicitly as sqrt iswap and inverse sqrt iswap.

Reply:
You are correct!  I tried to be too fancy for my own good.  I just made all the checks separate and explicit and added tests for the -7pi/4 and 7pi/4